Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -18,17 +18,21 @@
 AXUCode::AXUCode(DSPHLE* dsphle, u32 crc) : UCodeInterface(dsphle, crc), m_cmdlist_size(0)
 {
   INFO_LOG(DSPHLE, "Instantiating AXUCode: crc=%08x", crc);
-  m_mail_handler.PushMail(DSP_INIT);
-  DSP::GenerateDSPInterruptFromDSPEmu(DSP::INT_DSP);
-
-  LoadResamplingCoefficients();
 }
 
 AXUCode::~AXUCode()
 {
   m_mail_handler.Clear();
 }
 
+void AXUCode::Initialize()
+{
+  m_mail_handler.PushMail(DSP_INIT);
```

Comment:
Yeah, let's try to not mix such a behavior change.
Isn't this change exactly what you did in PR #3756 ? Do you want me to rebase and to open a new PR for it?